include/oneapi/dpl/internal/distributed_ranges_impl/shp/algorithms/execution_policy.hpp

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <span>
+#include <sycl/sycl.hpp>
+#include <vector>
+
+namespace oneapi::dpl::experimental::dr::shp
+{
+
+struct device_policy
```

Comment:
I understand this is in a separate namespace but I wonder if we want to overload the term `device_policy` 
(`oneapi::dpl::execution::device_policy`) 
Should this be a `distributed_device_policy`?

Reply:
renamed